prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-email-view/notification-email-view.component.html

```diff
@@ -11,14 +11,14 @@
     {{(editable) ? "Editing" : "Preview"}}
   </ng-container>
 
-  <button *ngIf="(Role.SUPER_ADMIN | inRole) && !editable"
+  <button *ngIf="((Role.SUPER_ADMIN | inRole) || (Role.MAINTENANCE | inRole)) && !editable"
           mat-icon-button
           matTooltip="Edit Email Template"
           (click)="toggleEdit(true)">
     <mat-icon>edit</mat-icon>
   </button>
 
-  <button *ngIf="(Role.SUPER_ADMIN | inRole) && editable"
+  <button *ngIf="((Role.SUPER_ADMIN | inRole) || (Role.MAINTENANCE | inRole)) && editable"
```

Comment:
Do we need the check for role here?
This button is inside a dialogue that shows after clicking `Edit` which would not show if the role check does not evaluate to true to display the `Edit` button in the first place (line 14)

Reply:
For consistency I would make it explicit.  Arguably this should be a single button that changes its configuration based on edit-ability instead of two buttons that create redundant markup in which case it would be applied to "both" as a single element.